app/views/fragments/global/footer.scala.html

```diff
@@ -23,8 +33,13 @@
                 You may also find help in our
                 <a href="@maybeProduct.getOrElse(Digipack).faqHref"
                    title="Frequently Asked Questions">
-                   Frequently Asked Questions.
+                   Frequently Asked Questions
                 </a>
+                and the @maybeProduct.getOrElse(Digipack).productType
```

Comment:
The footer is shared among the products so we need to think about what to do for the other products when referencing the digital pack T&Cs. Current behaviour uses this mapping https://github.com/guardian/subscriptions-frontend/blob/master/app/views/support/PlanOps.scala#L131